epersons.md

```diff
@@ -44,6 +44,7 @@ the replace operation `[{ "op": "replace", "path": "/netid", "value": "newNetId"
 ```json
   "netid": "newNetId",
 ```
+#### This operation can be performed by administrators and by the authenticated user.
```

Comment:
This is correct but as we are adding this details for the password I would like to have a similar indication for the previous replace operations as well that should be all reserved only to the administrator.
If possible please add also an example to change the email

Reply:
I added the email patch as an operation that can be performed by administrators and the authenticated user. I will change this if this operation needs to be restricted to administrators.

Reply:
Updated contract to say the email patch operation is limited to administrators.

Reply:
yes, this is better. The user is currently not allowed to change the email herself and introduce this feature could be more expensive than what it looks at a first glance as we should have a way to validate the email